js/augurs-prophet-js/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! JS bindings for the Prophet model.
-use std::{collections::HashMap, num::TryFromIntError};
-
 use js_sys::Float64Array;
 use serde::{Deserialize, Serialize};
+use std::num::NonZeroU32;
+use std::{collections::HashMap, num::TryFromIntError};
```

Comment:
Could you move these `std` imports back to a separate group? I've tried to keep them apart from third party imports across the project. (Also if you could group them into a single `use` statement that would be great!)